commit c6ebdda20e4f2426569b6dc47ae9c309ad638937 (refs/changes/72/4572/2)
Author: Rui Loura <rui@joyent.com>
Date:   2018-07-25T20:15:06+00:00 (1 year, 2 months ago)
    
    TRITON-607 cn-agent should be aware of alternate admin nictags (bump version)
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/package.json b/package.json
index 4508124..541c39f 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cn-agent",
   "description": "Triton Compute Node Agent",
-  "version": "2.2.1",
+  "version": "2.3.0",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
